1/src/components/auth/Login.vue

```diff
@@ -2,87 +2,102 @@
   <div class="inputContainer" style="display:right">
     <h3 v-if="loggedIn === false">Please enter your login details</h3>
     <h3 v-else>Hello User</h3>
-      <form action="#" @submit.prevent="">
-        <input type="email" placeholder="example@email.de" v-model="form.email" />
-        <input type="password" placeholder="******" v-model="form.password" />
-        <button type="submit" @click="login">sign in</button>
-        <!-- <button v-else type="submit" @click="logout">sign out</button> -->
+    <form action="#" @submit.prevent>
+      <input type="email" placeholder="example@email.de" v-model="form.email" />
+      <input type="password" placeholder="******" v-model="form.password" />
+      <button type="submit" @click="login">sign in</button>
+      <!-- <button v-else type="submit" @click="logout">sign out</button> -->
     </form>
   </div>
 </template>
 
 <script>
-import gql from 'graphql-tag'
+import gql from "graphql-tag";
 
 export default {
-    name: 'login',
-  props: {
-  },
+  name: "login",
+  props: {},
   methods: {
-      async login(){
-         await this.$apollo.mutate({
-      mutation: gql`mutation ($email: String!, $password: String!) {
-        login(email: $email, password: $password) {
-          id
+    async login() {
+      console.log("Login mutation started");
+      const data = await this.$apollo.mutate({
+        mutation: gql`
+          mutation($email: String!, $password: String!) {
+            login(email: $email, password: $password) {
+              jwt
+            }
+          }
+        `,
+        // Parameters
+        variables: {
+          email: this.form.email,
+          password: this.form.password
         }
-      }`,
-      // Parameters
-      variables: {
-         email: this.form.email,
-        password: this.form.password
-      },
-    }).then((data) => {
-                    if (data.data.login != null || data.data.login!= undefined) {
-                    this.loggedIn = true;
-                    this.id = data.data.login.id;
-                    }
-                    
-                }).catch((error) => {
-                    return error;
-                })
-      },
-      // async logout(){
-      //   await this.$apollo.mutate({
-      //     mutation: gql`mutation ($id: String!) {
-      //   logout(id: $id) {
-      //     id
-      //   }
-      // }`,
-      // // Parameters
-      // variables: {
-      //    id: this.id,
-      // },
-      //   }).then((data) => {
-      //               if (data.data.logout != null || data.data.logout!= undefined) {
-      //               this.loggedIn = false;
-      //               this.id = '';
-      //               }
-                    
-      //           }).catch((error) => {
-      //               return error;
-      //           })
-      // }
+      });
+      console.log("Before if");
+      if (data.data.login != null || data.data.login != undefined) {
```

Comment:
```suggestion
      if (!login) {
```

See above :point_up: how I assigned the local variable `login`.